### DISCUSSION TITLE
article_giscus_astro/

### DISCUSSION THREAD

**Discussion by giscus** [TheZalRevolt/TheZalDev_Blog#32] [Announcements]
# article_giscus_astro/

A little tutorial on how to integrate giscus on your Astro Blog

https://thezal.dev/article_giscus_astro/

**Comment by AmanVarshney01** [comment]
thanks

**Comment by zhenyou-nide** [comment]
When entering the page for the first time (blog theme is light), it seems like ![img](https://pic.imgdb.cn/item/669f1e91d9c307b7e98ece3f.png)

**Reply by zhenyou-nide** [reply]
![img](https://s1.locimg.com/2024/07/29/0c60553aa78ca.png)
